app/jobs/deposit_work_job.rb

```diff
@@ -131,16 +126,43 @@ def accession_or_persist_complete(druid:)
     end
   end
 
-  def update_globus_content # rubocop:disable Metrics/AbcSize
-    new_endpoint_client = endpoint_client_for(GlobusSupport.new_path(user: current_user, with_uploads_directory: true))
-    work_endpoint_client = endpoint_client_for(GlobusSupport.path(work:))
-    # rename if not persisted and globus?
-    if !work_form.persisted? && new_endpoint_client.exists? && !work_endpoint_client.exists?
-      new_endpoint_client.rename(new_path: GlobusSupport.path(work:, with_uploads_directory: true))
-    end
-    # remove permissions
-    work_endpoint_client.delete_access_rule
+  def update_globus_content
+    rename_globus_path if !work_form.persisted? && new_globus_path_exists? && !work_globus_path_exists?
+    remove_globus_permissions
   rescue StandardError => e
     raise unless e.message.include?('Access rule not found')
   end
+
+  def new_globus_path_exists?
+    GlobusClient.exists?(
+      user_id: user.email_address,
+      notify_email: false,
+      path: GlobusSupport.user_path(user: Current.user, with_uploads_directory: true)
+    )
+  end
+
+  def work_globus_path_exists?
+    GlobusClient.exists?(
+      user_id: user.email_address,
+      notify_email: false,
+      path: GlobusSupport.work_path(work:)
+    )
+  end
+
+  def rename_globus_path
+    GlobusClient.rename(
+      new_path: GlobusSupport.work_path(work:, with_uploads_directory: true),
+      user_id: user.email_address,
+      notify_email: false,
+      path: GlobusSupport.user_path(user: Current.user, with_uploads_directory: true)
```

Comment:
Without the GlobusSupport method renames, this call would have passed `GlobusSupport#path` as the `new_path` arg, and `GlobusSupport#new_path` as the `path` arg, which 😵